lib/engine/game/base.rb

```diff
@@ -577,12 +580,22 @@ def init_tiles
         self.class::TILES.flat_map do |name, val|
           if val.is_a?(Integer)
             count = val
-            count.times.map { |i| Tile.for(name, index: i) }
+            count.times.map do |i|
+              Tile.for(name,
+                       index: i,
+                       tile_reservation_blocks_others: self.class::TILE_RESERVATION_BLOCKS_OTHERS)
```

Comment:
do tile reservations only block for certain places, or they block every single one?

Reply:
In 1830 they block every slot on the tile until the token is played